lib/test_hook.rb

```diff
@@ -1,6 +1,13 @@
 class QsimTestHook < Mumukit::Templates::FileHook
   isolated true
 
+  attr_writer :renderer
+
+  def initialize(config=nil)
+    super config
+    @renderer = HtmlRenderer.new
+  end
```

Comment:
Instead of overriding `initialize`, in such cases where I don't actually need to pass an extra argument to the construct, I prefer to just declare a lazy attribute: 

```ruby
attr_writer :renderer
def renderer
   @renderer ||=   HtmlRenderer.new
end
```